### ISSUE TITLE
Subcommand IPA charts

### ISSUE THREAD

**Issue by xrotwang** [cldf/cldfbench#23]
Once we have a CLDF component for phoneme inventories, we can implement a subcommand `ipacharts`, creating either a big page `inventories.md` or multiple pages `inventories/<Language_ID>.md`, plotting the phoneme inventory of a language on an IPA chart.

**Comment by LinguList** [comment]
It would be good if we had exhaustive charts for the different sounds in CLTS. The normal IPA charts are typically not able to cover affricates, and the like. But it will require some work to think of the most frequent and most useful chart, I think. As an example, [this one](https://digling.org/edictor/plugouts/ipa_chart.html) originally comes from Wikipedia.

**Comment by xrotwang** [comment]
Since CLTS has features, we should be able to plug most phonemes into the
correct places, I think. Other than that, I'm not sure more exhaustive
would be good. Isn't it also good information if an inventory has empty
spots in the standard chats, and a lot of extra stuff?

Johann-Mattis List <notifications@github.com> schrieb am Sa., 25. Jan.
2020, 10:11:

**Comment by LinguList** [comment]
Yes and no. I would say: if you exclude affricates and nasal vowels, and aspiration, many normal features will look odd, as you can see in [phoible now](https://phoible.org/inventories/view/2398#tipa). But The extended chart from Wikipedia should be fine. @tresoldi might have an idea to organize a good general chart that is more expressive than current phoible? Or we just take the one I showed (and regularize with respect to clts)?
